lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -17,12 +17,28 @@
                     <% } %>
                     <% if (dateString && !is_unfulfilled_entitlement) { %>
                         <span class="run-period"><%- dateString %></span>
-                        <% if (user_entitlement && !is_unfulfilled_entitlement) { %>
+                        <% if (user_entitlement && !user_entitlement.expired_at && !is_unfulfilled_entitlement) { %>
                             <button class="change-session btn-link" aria-controls="change-session-<%-user_entitlement.uuid%>"> <%- gettext('Change Session')%></button>
                         <% } %>
                     <% } %>
                 </div>
-            </div>
+                <% if (user_entitlement && user_entitlement.expiration_date) { %>
+                    <div class="info-expires-at">
+                        <% if (is_unfulfilled_entitlement) { %>
+                            <% if (user_entitlement.expired_at) { %>
+                                <%- StringUtils.interpolate(gettext('You can no longer select a session as of {expiration_date}.'), {expiration_date: user_entitlement.expiration_date}) %>
```

Comment:
@HarryRein Can you pls clarify when this message appears? If it appears AFTER the expiration, can we make it the same as the other expired wording: "You can no longer select a session. Your final day to select a session was..."

Reply:
Yes, exactly - changed